### ISSUE TITLE
not found fonts in sveltekit 

### ISSUE THREAD

**Issue by lanzhixi** [open-source-labs/Svelvet#427]
![image](https://github.com/open-source-labs/Svelvet/assets/114967305/11ab8d8f-3d62-44da-88f5-e20651062746)
![image](https://github.com/open-source-labs/Svelvet/assets/114967305/1ad0d586-2b82-4e0f-94b8-35eba69dab2d)

**Comment by lanzhixi** [comment]
when version =8.0.0 ,problem is sovled

**Comment by briangregoryholmes** [comment]
Yeah, this is a mix of issues, unfortunately, and I hope to have it fixed this weekend. The fonts are there and, when you build the project, everything loads as expected, but it doesn't work in dev mode because of an issue out of our control. In all likelihood, I'll just remove the fonts from the package.

**Comment by briangregoryholmes** [comment]
Should be fixed in the latest version.
